packages/atomic/src/components/common/generated-answer/atomic-citation/atomic-citation.tsx

```diff
@@ -183,10 +183,10 @@ export class AtomicCitation {
             this.interactiveCitation.cancelPendingSelect()
           }
           stopPropagation={this.stopPropagation}
-          onMouseLeave={this.closePopover}
-          onMouseOver={this.delayedPopoverOpen}
-          onFocus={this.openPopover}
-          onBlur={this.closePopover}
+          onMouseLeave={this.debouncePopoverHide}
+          onMouseOver={this.showPopover}
```

Comment:
This is not just introducing the fix, this is changing the behaviour, the popover used to open after delay, now you are opening it right away.

- We should focus on just adding the fix and not changing an existing behaviour.